dashboard/app/models/concerns/levels/levels_within_levels.rb

```diff
@@ -73,6 +73,50 @@ def clone_child_levels(parent_level, new_suffix, editor_experiment: nil)
 
         return update_params
       end
+
+      # Create ParentLevelsChildLevel many-to-many relationships based on the
+      # contents of the specified levels' `contained_level_names` or `project_template_level_name` properties.
+      # Differentiated by level_kind (either contained or project template)
```

Comment:
nit; for clarity, please specify

```suggestion
      # Differentiated by level_kind (either ParentLevelsChildLevel::CONTAINED or ParentLevelsChildLevel::PROJECT_TEMPLATE)
```